src/ArduinoJson/MsgPack/BinaryValue.hpp

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+ARDUINOJSON_BEGIN_PUBLIC_NAMESPACE
+
+template <typename T>
+class BinaryValue {
```

Comment:
Supporting container is cool, but not worth the complexity.
Please simplify the class to support only `const void*` / `size_t`.